version.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE version SYSTEM "../../../lib/pkp/dtd/pluginVersion.dtd">
+<version>
+	<application>ojs-erc-plugin</application>
+	<type>plugins.generic</type>
+	<release>0.1.0.0</release>
+	<date>2019-05-15</date>
```

Comment:
Please start the notes for creating releases and document how this file needs to be updated.

Reply:
The notes are now available via a description in the README.md and the CHANGELOG.md. The corresponding commit can be found here: https://github.com/o2r-project/ojs-erc-plugin/pull/1/commits/eaea77a44ed699ff3321899fe6290f7d33fe0dd7